src/lib/libast/tm/tmxmake.c

```diff
@@ -98,7 +98,6 @@ Tm_t *tmxtm(Tm_t *tm, Time_t t, Tm_zone_t *zone) {
     tm->tm_mday = x + 1;
     tm->tm_nsec = tmxnsec(t);
     tmfix(tm);
-    n += 1900;
```

Comment:
This is correct but I feel compelled to point out that the preceding logic on line 94 which subtracts one and line 95 is needlessly obtuse. Don't change those lines because the benefit doesn't justify the risk. I just felt slightly nauseous when I reviewed the code to ensure removing this line was okay.